tests/MakersOfDenmark.WebAPI.Tests/Validator/RegisterMakerSpaceValidatorTest.cs

```diff
@@ -8,20 +8,45 @@
 
 namespace MakersOfDenmark.WebAPI.Tests.Validator
 {
-    public class RegisterMakerSpaceValidatorTest : IClassFixture<RequestHandlerFixture>
+    public class RegisterMakerSpaceValidatorTest
     {
-        private readonly RequestHandlerFixture _rhf;
+        private Fixture _fixture;
         private RegisterMakerSpaceValidator _registerMakerSpaceValidator;
-        public RegisterMakerSpaceValidatorTest(RequestHandlerFixture rhf)
+        public RegisterMakerSpaceValidatorTest()
         {
             _registerMakerSpaceValidator = new RegisterMakerSpaceValidator();
-            _rhf = rhf;
+            _fixture = new Fixture();
+        }
+        [Fact]
+        public void ValidModel__Fixture()
+        {
+            RegisterMakerSpace model = _fixture.Build<RegisterMakerSpace>().With(x => x.LogoUrl, "https://localhost").Create();
+            var result = _registerMakerSpaceValidator.TestValidate(model);
+            result.ShouldNotHaveAnyValidationErrors();
         }
 
+        /// <summary>
+        /// For illistrative purposes
+        /// </summary>
+        [Fact]
+        public void ValidName__WithoutFixture()
+        {
+            RegisterMakerSpace model = new RegisterMakerSpace {
+                AddressCity = "abc",
+                AddressCountry = "def",
+                AddressPostCode = "123",
+                AddressStreet = "streetgade",
+                AccessType = Domain.Enums.AccessType.Private,
+                ContactInfoEmail = "hest@abc.dk",
+                ContactInfoPhone = "456",
+            };
+            var result = _registerMakerSpaceValidator.TestValidate(model);
+            result.ShouldHaveValidationErrorFor(x => x.Name);
+        }
         [Fact]
-        public void ValidName()
+        public void ValidName_Fixture()
         {
-            RegisterMakerSpace model = _rhf.Fixture.Build<RegisterMakerSpace>().Without(x =>x.Name).Create();
+            RegisterMakerSpace model = _fixture.Build<RegisterMakerSpace>().Without(x => x.Name).Create();
             var result = _registerMakerSpaceValidator.TestValidate(model);
             result.ShouldHaveValidationErrorFor(x => x.Name);
         }
```

Comment:
Kan dette ikke laves om til en Theory hvor du gør på af ClassData?

Reply:
Ideen er god - men afvist - testen skal illustrere hvordan Fixture kan bruges - og testen er *for* simpel